lib/bindConnectors/interceptAfterSuccess.js

```diff
@@ -7,14 +7,33 @@ module.exports = function (message) {
 
 		message.model.afterSuccess = function (body, params, res) {
 
-			body = ( _.isArray(body) ? { results: body } : body );
+			body = ( modelAfterSuccess ? modelAfterSuccess(body, params, res) : body );
```

Comment:
Hmm... I think this needs to handle the possibility of `afterSuccess` being a promise as well:

![image](https://user-images.githubusercontent.com/2671525/33072378-331e59d6-ceb7-11e7-8d0a-c7f799e0c1b2.png)